keras/backend/tensorflow_backend.py

```diff
@@ -1274,6 +1274,13 @@ def log(x):
     return tf.log(x)
 
 
+def logsumexp(x, axis=None, keepdims=False):
+    '''Returns `log(sum(exp(x), axis=axis, keepdims=keepdims))` with improved numerical stability.
```

Comment:
The one-line docstring summary should fit within 80 chars.

Both added functions need a compliant docstring (`# Arguments`, `# Returns`).